examples/inference/benchmark_llama.py

```diff
@@ -58,12 +59,12 @@ def data_gen(batch_size: int = 4, seq_len: int = 512):
     return input_ids
 
 
-def print_details_info(model_config, args, whole_end2end):
+def print_details_info(model_config, args, whole_end2end, total_tokens_num):
```

Comment:
Grammatically, `total_tokens_num` should be `total_num_tokens` or `total_token_num`.